README.md

```diff
@@ -0,0 +1,242 @@
+Sawtooth or any of its sub-components including but not limited to Sawtooth
+Core, Sawtooth Supply Chain, Sawtooth Seth, the various Sawtooth SDKs, or the
+RFC process itself. What constitutes a "substantial" change is evolving based
+on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+  - Architectural changes
+  - Substantial changes to component interfaces
+  - New core features
+  - Backward incompatible changes
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does
+    not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism, trap
+    more errors, etc.)
+
+If you submit a pull request to implement a new feature without going through
+the RFC process, it may be closed with a polite request to submit an RFC first.
+
+
+### Sub-team specific guidelines
+[Sub-team specific guidelines]: #sub-team-specific-guidelines
+
+For more details on when an RFC is required for the following areas, please see
+the Sawtooth community's [sub-team] specific guidelines for:
+
+
+  - [core changes](core_changes.md)
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other project developers
+beforehand, to ascertain that the RFC may be desirable; having a consistent
+impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking
+the idea over on #sawtooth and proposing ideas to the Hyperledger Sawtooth
+mailing list (https://lists.hyperledger.org/mailman/listinfo/hyperledger-stl).
+
+As a rule of thumb, receiving encouraging feedback from long-standing project
+developers, and particularly members of the relevant [sub-team] is a good
+indication that the RFC is worth pursuing.
+
+
+## What the process is
+[What the process is]: #what-the-process-is
+
+In short, to get a major feature added to Sawtooth, one must first get the RFC
+merged into the RFC repository as a markdown file. At that point the RFC is
+"active" and may be implemented with the goal of eventual inclusion into Sawtooth.
+
+  - Fork the RFC repo [RFC repository]
+  - Copy `0000-template.md` to `text/0000-my-feature.md` (where "my-feature" is
+    descriptive. don't assign an RFC number yet).
```

Comment:
When is the number assigned?

Reply:
My current assumption is prior to merging, at the end of the FCP.

Reply:
Might be worth spelling it out, even though the Rust RFCS docs don't.